test-v13/build.gradle

```diff
@@ -24,6 +24,17 @@ android {
     sourceSets {
         main.java.srcDirs += "src/main/kotlin"
     }
+
+    testOptions {
```

Comment:
Do we need this? 👁

Reply:
I put this in case this happens again. Hopefully never...